### DISCUSSION TITLE
Marqueurs automatiques/Automatic markers

### DISCUSSION THREAD

**Discussion by ghost** [mytourbook/mytourbook#1317] [Q&A]
Hello,
First,I would like to say that I use Mytourbook with pleasure for planning long (and short) distance hikes and that it is really very practical.
However, I have a question regarding the markers:
Is it possible to automatically create markers? I am thinking for example of using segmentation every X kilometers or X hours then inserting the corresponding markers.
Does this function already exist elsewhere in the software, because I couldn't find it?
If not, is this a possible function in the future?
Thank you
Benoît

Bonjour,
Tout d'abord je voudrais dire que j'utilise Mytourbook avec plaisir pour la planification de randonnées longue (et courte) distance et qu'il est vraiment très pratique.
J'aurais toutefois une question concernant les marqueurs :
Est-il possible de créer automatiquement des marqueurs ? Je pense par exemple à l'utilisation de la segmentation tous les X kilomètres ou X heures puis insérer les marqueurs correspondants.
Cette fonction existe-t-elle déjà à un autre endroit du logiciel, car je n'ai pas trouvé ? ?
SI non, est-ce une fonction envisageable à l'avenir ?
Je vous remercie

**Comment by wolfgang-ch** [comment]
>Does this function already exist elsewhere in the software, because I couldn't find it?

This feature is not yet available

>If not, is this a possible function in the future?

Can you create a feature request for this, an english description is sufficient

**Reply by wolfgang-ch** [reply]
Created request https://github.com/mytourbook/mytourbook/issues/1319
